lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -534,7 +534,7 @@ protected function findIsolatedDiffTagsInOld($operation, $posInNew)
     protected function insertTag($tag, $cssClass, &$words)
     {
         while (true) {
-            if (count($words) == 0) {
+            if (!is_countable($words) || count($words) == 0) {
```

Comment:
`is_countable` is only available in php >= 7.3.0. In order to support older versions of PHP (right now composer.json is `"php": ">=5.3.3"`) we should use a different method of making sure $words is not null.